README.md

```diff
@@ -147,14 +154,34 @@ In general, every development board has the following basic features:
+- ### **Moving back to the vintage Bar-graph LED's and more switches**
+    - The previous edition used a pair of programmable switches and LEDs each but in 2020, the provision for a bar graph LED has been made. It has 10 LEDs out of which two are reserved for 5V and 3.3V voltage indication.
     -  So there are 8 programmable LEDs on the board. These LEDs multiplexed with directional pins of the two motor drivers to save pins.
     -  Directional pins? >> Every motor driver channel has two directional pins IN1, IN2. If IN1 is high and IN2 is low then motors move in a clockwise direction and there are 4 channels on the board, so 4 * 2 = 8 directional pins are multiplexed with 8 programmable LEDs.
     -  With 8 LEDs in hand, debugging get easier. Some examples - 
         1. According to the line sensor array (LSA) data, one can program 4 LEDs to turn on when the line sensor detects white  and turn off for black- line following debugging.
         2. If a motor is moving in a forward direction, the dedicated LEDs will be indicating IN1 is high and IN2 is low - motor control debugging.
+    - In the 2022 board, the LED Array was switched to a SMD LED Resistor pair. This resulted in conservation of space along with longer life. 
+    - Also, the 2022 board further conserves space by using SMD switches which have a smaller dimension of 4.5mmx4.5mm compared to their 6mmx6mm older counterparts, achieving the same functionality.
+
+- ### **Connectors**
+    - The earlier editions of the board used FRC (Flat Ribbon Cable) Connectors for connecting sensors with the SRA board. FRC Cables are formed by joining insulated wires in a flat shape forming the Ribbon shape which invariably means they take up a good amount of space.
+    - The 2022 board has replaced all FRC connectors with JST-PH Connectors, which are much more easily available and are designed to be space-efficient in high-density connection of internal wires such as ours. 
+    - These JST-PH connectors also have the added advantage of being compact, highly reliable and low in cost.
+
+| Previous FRC Connectors  |  Current JST-PH Connectors  | 
+|:----:|:-------:| 
+|<img width="150" height="150" src="/documentation/assets/frc.jpeg"> | ![jstPh_Connectors](https://user-images.githubusercontent.com/84874044/153403543-0e08c307-8752-45e2-b3d6-c5373742ea91.png)|
+
+- ### **Fuses**
+    - Bulky glass fuses in the earlier editions of the board were replaced by PTC or Positive Temperature Coefficient resettable fuses, enabling autorecovery from a temporary overload without any user intervention.
+    - A PTC fuse 'trips' by changing from a low resistance to a high resistance in order to limit current flow, which is much more desirable than a glass fuse that would melt open to interrupt the current flow and would have to be subsequently replaced.
```

Comment:
Mention that the resistance offered and the voltage drop found across the fuse increases with an increase in load current and also temperature of the fuse rises alongwith